.github/workflows/docs-pages.yml

```diff
@@ -0,0 +1,30 @@
+name: Docs Pages
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build Enc
+        run: echo "Nothing here."
+
+      - name: GitHub Pages
+        if: success()
+        uses: crazy-max/ghaction-github-pages@v2.3.0
+        with:
+          target_branch: enc
+          keep_history: true
```

Comment:
It is useful for comparing different versions of generated data.

Reply:
There's no _generated_ data.
tool.make_inet.py only replaces a couple of MSDN links and the index page, otherwise it's exactly the same as the source.

Reply:
Styles sheets are also different.

In any case being able to inspect the history of the web site is useful to track changes to markup. I planned to add Lua to online reference.

Or do you worry that the web site history will take too much space?

Reply:
Just reluctant about spamming the repository with history that is already fully preserved in enc/.

Reply:
The history is in separate branch. It might be possible to commit into separate repo. And IIRC new commits are only created when there is a change, so emails will be rare.

Reply:
> It might be possible to commit into separate repo

I like this option. What is needed? Just a new repo under FarGroup/?

Reply:
New repo and generated personal access token to use in `GH_PAT` environment variable.

https://github.com/crazy-max/ghaction-github-pages#environment-variables

Reply:
Actually the whole `/enc` content could be moved to a new repo, because it doesn't use Far sources.

Reply:
> New repo and generated personal access token to use in GH_PAT environment variable.

https://github.com/FarGroup/api

Reply:
Should work now.

Reply:
That breaks previews in forks, but I can fix that later/